int16 NFC_TAG_TYPE1TAG=100

# Type2 Tags
int16 NFC_TAG_TYPE2TAG=200
int16 NFC_TAG_TYPE2NXPMIFAREULTRALIGHT=201
int16 NFC_TAG_TYPE2NXPMIFAREULTRALIGHTC=202
int16 NFC_TAG_TYPE2NXPNTAG210=203
int16 NFC_TAG_TYPE2NXPNTAG212=204
int16 NFC_TAG_TYPE2NXPNTAG215=205
int16 NFC_TAG_TYPE2NXPNTAG216=206
int16 NFC_TAG_TYPE2NXPMIFAREULTRALIGHTEV1=207

# Type3 Tags
int16 NFC_TAG_TYPE3TAG=300
int16 NFC_TAG_FelicaStandard=301
int16 NFC_TAG_FelicaMobile=302
int16 NFC_TAG_FelicaLite=303
int16 NFC_TAG_FelicaLiteS=304
int16 NFC_TAG_FelicaPlug=305

# Type4 Tags
int16 NFC_TAG_TYPE4TAG=400
int16 NFC_TAG_TYPE4ATAG=401
int16 NFC_TAG_TYPE4BTAG=402

int16 tag_type
NDEFRecord[] ndef_records
string identifier
